pkg/gameservers/gameservercounter.go

```diff
@@ -44,19 +44,21 @@ type AllocationCounter struct {
 // NodeCount is just a convenience data structure for
 // keeping relevant GameServer counts about Nodes
 type NodeCount struct {
-	ready     int64
-	allocated int64
+	// Ready is ready count
+	Ready int64
+	// Allocated is allocated out
+	Allocated int64
 }
 
-// NewAllocationCounter returns a new AllocationCounter
-func NewAllocationCounter(
+// NewGameServerCounter returns a new GameServerCounter
+func NewGameServerCounter(
 	kubeInformerFactory informers.SharedInformerFactory,
-	agonesInformerFactory externalversions.SharedInformerFactory) *AllocationCounter {
+	agonesInformerFactory externalversions.SharedInformerFactory) *GameServerCounter {
 
 	gameServers := agonesInformerFactory.Stable().V1alpha1().GameServers()
 	gsInformer := gameServers.Informer()
 
-	ac := &AllocationCounter{
+	ac := &GameServerCounter{
```

Comment:
nit name also could be changed
```suggestion
	gc := &GameServerCounter{
```